### PR TITLE
Use interpolateUrl in logo src config

### PR DESCRIPTION
**Pull request by brandones**
This will allow the config to refer to `${openmrsBase}`, which will allow servers to use `/openmrs` or `/mirebalais` as appropriate.

Also a little bit of miscellaneous unrelated refactoring. Should be pretty easy to tell what's what.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -82,7 +82,7 @@
     "pretty-quick": "^3.1.1",
     "ramda": "^0.27.1",
     "react-i18next": "^11.3.1",
-    "rxjs": "^7.4.0",
-    "typescript": "^4.4.3"
+    "rxjs": "^6.6.7",
```
Comment (mogoodrich) [inline package.json:85]: Intentionally downgrading?

**Review by mogoodrich — APPROVED** [review]
LGTM!

**Review by mseaton — APPROVED** [review]
Assuming interpolateUrl function already exists somewhere and this is just updating to use it, seems fine.

File: package.json (hunk 1 above)
Comment (brandones) [reply package.json:85]: Yeah, `@openmrs/esm-framework` depends on 6.x. Having 7.x in there introduced a type conflict.

**Pull request merged**